pkg/driverbuilder/builder/builders.go

```diff
@@ -12,16 +12,16 @@ import (
 const DriverDirectory = "/tmp/driver"
 
 // ModuleFileName is the standard file name for the kernel module.
-const ModuleFileName = "falco.ko"
+const ModuleFileName = "module.ko"
```

Comment:
I'm not getting why this change, Luca.

Not that it is a huge problem, the patch looks good!

Reply:
It isn't really necessary, but I wanted to catch cases where the `falco` string is hardcoded so when running the script in the container it wouldn't "accidentally" work. Also, it's symmetrical with `probe.o`. No big deal either way though, let me know if you feel like it's better to leave it `falco` and I'll revert this change.

Reply:
Naaa, it makes sense. I understood your reasoning now.